Common/DocxFormat/Source/XlsxFormat/Common.cpp

```diff
@@ -48,6 +48,11 @@ char* gcvt(double x, int n, char* b)
 #define _gcvt gcvt
 #endif
 
+#ifdef __FreeBSD__
+#define _gcvt(x,n,b) sprintf(b, "%.17g", x)

```

Comment:
Android uses:
```
char* gcvt(double x, int n, char* b) {
    sprintf(b, "%.*g", n, x);
    return b;
}
X2
```
see XtConverter/build/Android/libx2t/src/main/cpp/workaround/gcvt/gcvt.c

Reply:
it's nearly the same result..